packages/components/button-bar/src/button-bar.scss

```diff
@@ -1,46 +1,36 @@
 :host {
-  // TODO: remove this "hack" when style queries are possible.
-  --sl-ON: initial; // property gets fallback value
-  --sl-OFF: var(--sl-button-bar-direction-row);
```

Comment:
Can you please also make changes of the button bar usage in the dialog?
In the mobile version of the dialog we want to have buttons reversed.

<img width="572" alt="image" src="https://github.com/user-attachments/assets/55eee3d2-43d4-4767-96f1-10500fbfd73b" />